models/experimental/functional_unet/tt/unet_shallow_ttnn.py

```diff
@@ -3,76 +3,18 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import torch
+import torch.nn as nn
 import ttnn
 
 import tt_lib as ttl
```

Comment:
Can we not mix and match libraries? If you want to use ttl, just use `ttnn.experimental`